code/mob/living/silicon/robot_oil.dm

```diff
@@ -0,0 +1,11 @@
+/datum/lifeprocess/robot_oil
+	//moved from robot.dm proc/process_oil to this life process
+	process(var/datum/gas_mixture/environment)
+		if(robot_owner)
+			var/mult = get_multiplier()
+			robot_owner.oil -= 1*mult
```

Comment:
why multiply 1 by it? useless

Reply:
right, wasn't thinking about that at the time, was trying to get everything around it to work.